### PR TITLE
Translation Improvement

### PR DESCRIPTION
**Pull request by nicobubulle**

### CODE REVIEW

File: src/client/components/GameEnd.vue [hunk 1]
```diff
@@ -22,9 +22,9 @@
               <div v-else>
                   <div class="game_end_fail">
                       <h2 v-i18n>Sorry, you lose.</h2>
-                      <div class="game_end_notice" v-i18n>
-                          Next time you will get more luck!<br>
-                          Also, take into account these small hints to win:
+                      <div class="game_end_notice">
+                        <span v-i18n>Next time you will get more luck!</span><br>
```
Comment (kberg) [inline src/client/components/GameEnd.vue:26]: I have been thinking about rewriting these to be more clear, and which point your translations might be out of date. Would it be a problem if you translated twice?

**Review by kberg — COMMENTED** [review]
Looks great. I just have that one question.

File: src/client/components/GameEnd.vue (hunk 1 above)
Comment (nicobubulle) [reply src/client/components/GameEnd.vue:26]: No worry about that, I still have a lot of work with French translation. I will translated it again.

**Review by kberg — APPROVED** [review]

File: src/client/components/GameEnd.vue (hunk 1 above)
Comment (kberg) [reply src/client/components/GameEnd.vue:26]: Great. Thanks for this!

**Pull request merged**